internal/otelutil/otel_tracing.go

```diff
@@ -106,17 +116,37 @@ func SetupTracing(cfg config.OTelTracingConfig, resourceLibrary string, resource
 		}
 	}
 
-	otelshutdown, err := otelconfig.ConfigureOpenTelemetry(
-		otelconfig.WithExporterProtocol(protocol),
-		otelconfig.WithServiceName(cfg.Dataset),
-		otelconfig.WithTracesExporterEndpoint(apihost),
-		otelconfig.WithMetricsEnabled(false),
-		otelconfig.WithTracesEnabled(true),
-		otelconfig.WithSampler(samplers.TraceIDRatioBased(sampleRatio)),
-		otelconfig.WithHeaders(headers),
+	tlsconfig := &tls.Config{}
+	secureOption := otlptracehttp.WithTLSClientConfig(tlsconfig)
+	exporter, err := otlptrace.New(
+		context.Background(),
+		otlptracehttp.NewClient(
+			secureOption,
+			otlptracehttp.WithEndpoint(apihost.Host),
+			otlptracehttp.WithHeaders(headers),
+			otlptracehttp.WithCompression(otlptracehttp.GzipCompression),
+		),
 	)
 	if err != nil {
-		log.Fatalf("failure configuring otel: %v", err)
+		log.Fatalf("failure configuring otel trace exporter: %v", err)
+	}
+
+	otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(
```

Comment:
Why do we need this? I don't think we want refinery to do baggage propagation or anything having to do with b3?

Reply:
I copied from otel-config-go lol